src/main.c

```diff
@@ -85,6 +85,7 @@ PPPD_USAGE \
 "                    [--user-cert=<file>] [--user-key=<file>]\n" \
 "                    [--use-syslog] [--trusted-cert=<digest>]\n" \
 "                    [--persistent=<interval>] [-c <file>] [-v|-q]\n" \
```

Comment:
Shouldn't line 88 replace line 87 here? But maybe the line becomes too long then and should be split.